src/components/Conversation/index.tsx

```diff
@@ -99,6 +100,7 @@ export default function Conversation() {
             )}
             {publisher === PublisherState.NeedsDelegation && <DelegationBox />}
             {publisher === PublisherState.NeedsPermission && <PermitBox />}
+            {publisher === PublisherState.NeedsTokenGatedPermission && <TokenGatedBox />}
```

Comment:
Since one gives up their `grant` capacity after deploying the tokenGate the `NeedsPermission` and `NeedsTokenGatedPermission` should be mutually exclusive, but both require the delegation to be enabled to run